src/web-components/ConfirmPopup/index.ts

```diff
@@ -0,0 +1,91 @@
+import { LitElement, html, css } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import { ConfirmPopupStyles } from './styles';
+/**
+ * `MainPopup` is a custom popup component for displaying messages and handling user actions.
+ * 
+ * ### Properties:
+ * - **open**: (boolean) Indicates whether the popup is open or closed.
+ * - **message**: (string) The primary message to display in the popup.
+ * - **secondMessage**: (string) The secondary message to display in the popup.
+ * - **buttonConfirm**: (string) The label for the confirm button.
+ * - **buttonCancel**: (string) The label for the cancel button.
+ * - **confirmCallback**: (Function|null) A callback function to be called when the confirm button is clicked.
+ * 
+ * ### Example:
+ * ```html
+ * <main-popup 
+ *     open="true" 
+ *     message="Are you sure?" 
+ *     secondMessage="This action cannot be undone."
+ *     buttonConfirm="Confirm" 
+ *     buttonCancel="Cancel" 
+ *     .confirmCallback=${() => console.log('Confirmed!')}>
+ * </main-popup>
+ * ```
+ */
+@customElement('confirm-popup')
+export class ConfirmPopup extends LitElement {
+    static styles = ConfirmPopupStyles;
+    @property({ type: Boolean }) open = false;
+    @property({ type: String }) message = '';
+    @property({ type: String }) secondMessage = '';
+    @property({ type: String }) buttonConfirm = '';
+    @property({ type: String }) buttonCancel = '';
+    @property({ type: Function }) confirmCallback: (() => void) | null = null;
+    @property({ type: Function }) confirmCancel: (() => void) | null = null;
```

Comment:
`confirmCancel` is confusing. Maybe we call this `onCancel` and change `confirmCallback` to `onConfirm`